packages/standalone/install.sh

```diff
@@ -86,22 +88,17 @@ parse_args() {
     key="$1"
 
     case $key in
-      -d | --install-dir)
-        install_dir="$2"
-        shift # past argument
-        shift # past value
-        ;;
+      # -d | --install-dir)
+      #   install_dir="$2"
+      #   shift # past argument
+      #   shift # past value
+      #   ;;
       -s | --skip-shell)
         SKIP_SHELL="true"
         shift # past argument
         ;;
-      --force-install | --force-no-brew)
```

Comment:
nit: No more -d, -force-install any more? Are we losing func? I thought we are just updating `release` to `version`.

Reply:
they weren't used actually so I removed them.

and yeah I just removed things to be simpler right now and give us time to think what option we really want, nothing of this is documented and this feels like a use at your own risk, the standalone cli is explicitly marked as preview in the docs.